Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -479,6 +490,20 @@ extension ActivityStreamPanel: DataObserverDelegate {
             return succeed()
         }
     }
+    
+    func getPocketSites() -> Success {
+        var numItems: CGFloat = ASPanelUX.numberOfItemsPerRowForSizeClassIpad[self.traitCollection.horizontalSizeClass]
+        if UIInterfaceOrientationIsPortrait(UIApplication.shared.statusBarOrientation) {
+            numItems = numItems - 1
+        }
+        if self.traitCollection.horizontalSizeClass == .compact && UIInterfaceOrientationIsLandscape(UIApplication.shared.statusBarOrientation) {
+            numItems = numItems - 1
+        }
+        return pocketAPI.globalFeed(items: Int(numItems)).bindQueue(.main) { pStory in
```

Comment:
If you only fetch the needed amount of stories from the API, If you are in portrait you would see 3, Once you rotate there are 4. But you've only loaded 3 so there would be an empty spot. Instead always load the max which is 4 and you can avoid doing the math here.